src/aleph/vm/hypervisors/qemu/qemuvm.py

```diff
@@ -125,3 +125,7 @@ def send_shutdown_message(self):
                 logger.warning("unexpected answer from VM", resp)
             print("shutdown message sent")
             client.close()
+
+    async def teardown(self):
+        """Stop the VM, cleanup network interface and remove data directory."""
```

Comment:
The docstring is wrong
it's not actually doing any of that except stopping the vm.

Even on microvm it's not cleaning up the interface